sources/packages/backend/libs/test-utils/src/factories/msfaa-number.ts

```diff
@@ -40,12 +41,14 @@ export function createFakeMSFAANumber(
   },
   options?: {
     state?: MSFAAStates;
+    offeringIntensity?: OfferingIntensity;
   },
 ): MSFAANumber {
   const now = new Date();
   const dateOnly = getISODateOnlyString(now);
   const msfaaNumber = new MSFAANumber();
-  msfaaNumber.offeringIntensity = OfferingIntensity.fullTime;
+  msfaaNumber.offeringIntensity =
```

Comment:
Is this a direct assignment?

Reply:
`initialValues:Partial<MSFAANumber>`

Reply:
I believe that the idea was to have it added inside the options and named, as @dheepak-aot mentioned.